tests/data/jetstream_sg2210p.json

```diff
@@ -1,2560 +0,0 @@
-{
```

Comment:
How come you've removed this test data?

Reply:
... in worst case, i could restore test data , but port descr will be blank